### PR TITLE
WIP: Remove runner_version in target

### PR DESCRIPTION
**Pull request by gamoutatsumi**
In accordance with #159 change, `runner_version` come to be set by environment variables.

### CODE REVIEW

File: internal/config/init.go [hunk 1]
```diff
@@ -142,6 +144,23 @@ func LoadWithDefault() Conf {
...
+		ephemeralSupportVersion, err := version.NewVersion("v2.282.0")
+		if err != nil {
+			log.Panicf("failed to parse ephemeral runner version: %+v", err)
+		}
+		inputVersion, err := version.NewVersion(os.Getenv(EnvRunnerVersion))
+		if err != nil {
+			log.Panicf("failed to parse input runner version: %+v", err)
+		}
+		if !ephemeralSupportVersion.GreaterThan(inputVersion) {
+			log.Panicf("RUNNER_VERSION must be greater than %s", ephemeralSupportVersion)
```
Comment (whywaita) [inline internal/config/init.go:158]: myshoes also support the old version's less ephemeral mode.
you needn't panic here.

File: pkg/starter/scripts.go [hunk 2]
```diff
@@ -49,7 +50,7 @@ func (s *Starter) getSetupRawScript(ctx context.Context, target datastore.Target
 	if target.RunnerUser.Valid {
 		runnerUser = target.RunnerUser.String
 	}
-	runnerVersion, runnerTemporaryMode, err := datastore.GetRunnerTemporaryMode(target.RunnerVersion)
+	runnerVersion, runnerTemporaryMode, err := runner.GetRunnerTemporaryMode(s.runnerVersion)
```
Comment (whywaita) [inline pkg/starter/scripts.go:53]: runner_version in the bash script require `v2.xxx.yy` (format).
But `s.runnerVersion` is `latest` by default.

Please change to use the latest version in execute timing.

**Review by whywaita — CHANGES_REQUESTED** [review]

File: internal/config/init.go (hunk 1 above)
Comment (gamoutatsumi) [reply internal/config/init.go]: I deleted that, but I think need to panic when version is invalid.

File: pkg/starter/scripts.go (hunk 2 above)
Comment (gamoutatsumi) [reply pkg/starter/scripts.go:53]: Added checking for latest version.

**Pull request merged**